src/lib/age-warning.ts

```diff
@@ -3,41 +3,41 @@ export const getAgeWarning = (
     webPublicationDate: string,
 ): string | undefined => {
     const isNews = tags.some(t => t.id === 'tone/news');
+    const isOpinion = tags.some(t => t.id === 'commentisfree/commentisfree');
```

Comment:
i assume this is beyond your control, but `commentisfree` doesn't mean much to me

Reply:
Naming things...